pom.xml

```diff
@@ -18,6 +18,8 @@
     <nexus.url>https://oss.sonatype.org</nexus.url>
     <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
     <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
+    <version.maven.base>3.8.6</version.maven.base>
```

Comment:
I wonder if bumping the version prohibits us from running the Bloop plugin with earlier versions of maven?

Reply:
I don't _think_ so since these are now all provided. So for an older version of Maven, the version being used will be the version that this ends up being from their Maven. I didn't find a ton of open source projects using Scala + Maven so I haven't tested it heavily, but it did work on the ones that I tested. Although all of them were using a newer-ish Maven version. 3.6.1 was released in [2019](https://maven.apache.org/docs/history.html) so it's quite old, and the upgrade path on Maven in the 3.x I believe isn't too difficult.

One thing we can do is if we feel comfortable merging this in, I can release a RC canidate and then ping a couple people I know that use Maven at their workspace and see if they are willing to run

```
mvn ch.epfl.scala:bloop-maven-plugin:2.0.0-RC1:bloopInstall -DdownloadSources=true
```

And verify that it works before doing a final release. wdyt?